docs/examples/plot_lcppn_explainer.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+"""
+============================================
+Explaining Local Classifier Per Parent Node
+============================================
+
+A minimalist example showing how to use HiClass Explainer to obtain SHAP values of LCPPN model.
+A detailed summary of the Explainer class has been given at Algorithms Overview Section for :ref:`Hierarchical Explainability`.
+"""
+import numpy as np
+from sklearn.ensemble import RandomForestClassifier
+from hiclass import LocalClassifierPerParentNode, Explainer
+
+# Define data
+X_train = np.array(
+    [
+        [40.7, 1.0, 1.0, 2.0, 5.0, 2.0, 1.0, 5.0, 34.3],
+        [39.2, 0.0, 2.0, 4.0, 1.0, 3.0, 1.0, 2.0, 34.1],
+        [40.6, 0.0, 3.0, 1.0, 4.0, 5.0, 0.0, 6.0, 27.7],
+        [36.5, 0.0, 3.0, 1.0, 2.0, 2.0, 0.0, 2.0, 39.9],
+    ]
+)
+X_test = np.array([[35.5, 0.0, 1.0, 1.0, 3.0, 3.0, 0.0, 2.0, 37.5]])
+Y_train = np.array(
+    [
+        ["Gastrointestinal", "Norovirus", ""],
+        ["Respiratory", "Covid", ""],
+        ["Allergy", "External", "Bee Allergy"],
+        ["Respiratory", "Cold", ""],
+    ]
+)
```

Comment:
is it possible to create a function to download and load your platypus dataset? if it is not too large and can be executed quickly on read the docs, I would suggest doing something similar to this example from [sklearn](https://scikit-learn.org/stable/modules/generated/sklearn.datasets.load_iris.html#sklearn.datasets.load_iris):

```python
>>> from sklearn.datasets import load_iris
>>> data = load_iris()
>>> data.target[[10, 25, 50]]
array([0, 0, 1])
>>> list(data.target_names)
['setosa', 'versicolor', 'virginica']
```

Also maybe you can add some figures here like the ones you have on colabs.

Reply:
I have updated the `plot_lcppn_explainer.py` example to fetch a CSV file (containing ~800 entries). The example now uses the platypus diseases dataset to generate the shape values. One visualization example has also been included.

You can check the rendered example on Read the Docs [here](https://hiclass--106.org.readthedocs.build/en/106/auto_examples/plot_lcppn_explainer.html#sphx-glr-auto-examples-plot-lcppn-explainer-py).

Resolved by https://github.com/scikit-learn-contrib/hiclass/pull/106/commits/820c1a52370d8e37d36d285ef059a62e789b1850.